docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -1,86 +1,111 @@
 ---
 id: eks-eksctl
-title: "Deploy an EKS cluster with eksctl"
+title: "Deploy an EKS cluster with eksctl (quickstart)"
```

Comment:
I feel like "quickstart" is a little awkward here, as most of our docs could be categorized as "quickstart?" WDYT @conceptualshark?